### ISSUE TITLE
LFTP4WIN Installation failed

### ISSUE THREAD

**Issue by Zemur11** [userdocs/lftp-windows#21]
Hello, I am having trouble installing LFTP4WIN, I ran as administrator and cygwin seemed to install fine, but it gets here and fails: 'C:\Windows\system32>"D:\Programs\LFTP4WIN\system\bin\mkgroup.exe" -c   1>system/etc/group  || goto :fail
The system cannot find the path specified.'

I guess it's trying to access or create a folder path and failing but I don't understand why. I've tried installing from a different drive, as well as manually creating the 'group' folder under system/etc/, which of course did not help. Any advice is much appreciated!

```
C:\Windows\system32>"D:\Programs\LFTP4WIN\system\tmp\setup-x86_64.exe" --no-admin  --site "https://cygwin.mirrors.hoobly.com"  --root "D:\Programs\LFTP4WIN\system"  --local-package-dir "D:\Programs\LFTP4WIN\system\.pkg-cache"  --no-shortcuts  --no-desktop  --delete-orphans  --upgrade-also  --no-replaceonreboot  --quiet-mode  --packages wget,ca-certificates,gnupg,bsdtar,bash-completion,curl,lftp,ssh-pageant,openssh,openssl,sshpass,procps-ng    || goto :fail
Starting cygwin install, version 2.915
User has backup/restore rights
User has symlink creation right
Current Directory: D:\Programs\LFTP4WIN\system\.pkg-cache
root: D:\Programs\LFTP4WIN\system system
Selected local directory: D:\Programs\LFTP4WIN\system\.pkg-cache
net: Preconfig
site: https://cygwin.mirrors.hoobly.com/
solving: 12 tasks, update: yes, use test packages: no
solving: 0 tasks, update: no, use test packages: no
Augmented Transaction List: is empty
Changing gid back to original
running: D:\Programs\LFTP4WIN\system\bin\dash.exe "/etc/postinstall/0p_000_autorebase.dash"
running: D:\Programs\LFTP4WIN\system\bin\dash.exe "/etc/postinstall/0p_update-info-dir.dash"
running: D:\Programs\LFTP4WIN\system\bin\dash.exe "/etc/postinstall/zp_man-db-update-index.dash"
Changing gid to Administrators
Ending cygwin install

C:\Windows\system32>if "yes" == "yes" (rd /s /q "D:\Programs\LFTP4WIN\system\.pkg-cache" )

C:\Windows\system32>(
echo # /etc/fstab
 echo # IMPORTANT: this files is recreated on each start by LFTP4WIN-terminal.cmd
 echo #
 echo #    This file is read once by the first process in a Cygwin process tree.
 echo #    To pick up changes, restart all Cygwin processes.  For a description
 echo #    see https://cygwin.com/cygwin-ug-net/using.html#mount-table
 echo #
 echo none /cygdrive cygdrive binary,noacl,posix=0,sparse,user 0 0
) 1>"D:\Programs\LFTP4WIN\system\etc\fstab"

C:\Windows\system32>"D:\Programs\LFTP4WIN\system\bin\mkgroup.exe" -c   1>system/etc/group  || goto :fail
The system cannot find the path specified.

C:\Windows\system32>if exist "" (del "" )

C:\Windows\system32>echo.


C:\Windows\system32>echo ###########################################################
###########################################################

C:\Windows\system32>echo # Installing [LFTP4WIN Portable] FAILED!
# Installing [LFTP4WIN Portable] FAILED!

C:\Windows\system32>echo ###########################################################
###########################################################

C:\Windows\system32>echo.


C:\Windows\system32>timeout /T 60

Waiting for 49 seconds, press a key to continue ...
```

**Comment by userdocs** [comment]
What OS?

**Comment by Zemur11** [comment]
Hello, I am using Windows 10 64bit

**Comment by userdocs** [comment]
Don't run the installer as admin.

**Comment by Zemur11** [comment]
When I run it normally without admin rights, the cmd window pops up for just a split second and then closes, and nothing appears to happen

**Comment by userdocs** [comment]
The script does NOT need admin rights. Maybe there  is something stopping it on your side?

**Comment by Zemur11** [comment]
I finally got it working! I changed the directory to C:\LFTP4WIN instead of on D: drive, and also ran the installer by navigating to the folder in cmd and using 'start LFTP4WIN-installer.cmd', and it ran without admin rights, and without closing immediately. 

After that, it still didn't work, giving this error: 
`'bitsadmin' is not recognized as an internal or external command, operable program or batch file.`

for whatever reason, I had to change the code to specify the location of bitsadmin like this:
```
C:\Windows\System32\bitsadmin.exe /transfer cygwin /download /priority FOREGROUND /DYNAMIC 
C:\Windows\System32\bitsadmin.exe /transfer lftp4win /download /priority FOREGROUND /DYNAMIC 
```

I guess bitsadmin is not added as PATH Environment Variable? Not sure, but it's working perfectly now.
